coverage/htmlfiles/index.html

```diff
@@ -48,6 +48,13 @@ <h1>{{ title|escape }}:
             <input id="filter" type="text" value="" placeholder="filter..." />
         </form>
     </div>
+
+    <nav>
```

Comment:
I don't think I would use a `<nav>` element here.  The information here isn't navigation.